src/azure-cli-testsdk/azure/cli/testsdk/utilities.py

```diff
@@ -34,3 +35,29 @@ def get_active_api_profile():
     except CloudNotRegisteredException:
         init_known_clouds()
         return get_active_cloud().profile
+
+
+@contextmanager
+def force_progress_logging():
+    from six import StringIO
+    import logging
+    from azure.cli.core.commands import logger as cmd_logger
+
+    # register a progress logger handler to get the content to verify
+    test_io = StringIO()
+    test_handler = logging.StreamHandler(test_io)
+    cmd_logger.addHandler(test_handler)
+    old_cmd_level = cmd_logger.level
+    cmd_logger.setLevel(logging.INFO)
+
+    # this tells progress logger we are under verbose, so should log
+    az_logger = logging.getLogger('az')
+    old_az_level = az_logger.handlers[0].level
+    az_logger.handlers[0].level = logging.INFO
```

Comment:
I'm not sure how stable this would be. The first handler could be file handler or a colorized stream handler.